modules/integration_azure-container-instance/conf/readme.yaml

```diff
@@ -0,0 +1,11 @@
+documentations:
+  - name: Azure Monitor metrics
+    url: 'https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-supported'
```

Comment:
```suggestion
    url: 'https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-supported#microsoftcontainerinstancecontainergroups'
```

Reply:
why not but if you apply this suggestion you should remove this link from the `source_doc`, not useful to duplicate the information.

Keep it as it is makes sens for me : 
- related doc list (here) uses the azure monitor source, global documentation
- additional info about source includes a more specific link to the right resource (source_doc)

not obvious in the config file but fully logic on the rendered reamde.md